pkg/cloudprovider/provider/anexia/types/types.go

```diff
@@ -17,9 +17,22 @@ limitations under the License.
 package types
 
 import (
+	"time"
+
 	providerconfigtypes "github.com/kubermatic/machine-controller/pkg/providerconfig/types"
 )
 
+const (
+	AnxTokenEnv = "ANXCLOUD_TOKEN"
+
+	CreateRequestTimeout = 15 * time.Minute
+	GetRequestTimeout    = 1 * time.Minute
+	DeleteRequestTimeout = 1 * time.Minute
+
+	VmxNet3NIC       = "vmxnet3"
```

Comment:
@erdii should we support different NIC types than this? If not, then we should not expose this to machine deployment template because we don't want to expose this to user.